relational/table.py

```diff
@@ -265,6 +270,31 @@ def _gen_facts(self, row, proj):
             row[self.index(self.PERMIT_AUTHORITY_ID)] = proj.field(
                 'fk', MOHCDInclusionary.NAME)
 
+    def _estimate_bmr(self, net):
+        """Estimates the BMR we project a project to have.
+
+        This exists because currently all/most projects in ppts have nothing
+        specified for their affordable unit counts, but we can provide
+        a rough estimate of what we expect the project to have when it gets
+        entitled.
+
+        net: a (string) net unit count
+
+        Returns: a non-empty string
+        """
+        # TODO: this logic can get pretty complicated if needed, but right
```

Comment:
Nit: This seems like the type of thing we shoudl track somewhere (maybe a github issue that is just "Improve algorithm for predicting xyz"). We're getting a little TODO heavy, and feels like having this tracked is a good way to make sure someone (after we're gone) actually sees it?

Reply:
done